client/components/Medical-Contact/ContactPage.vue

```diff
@@ -0,0 +1,51 @@
+<!-- Page for medical contacts management -->
+<!-- User should be authenticated in order to see this page -->
+
+<template>
+  <main>
+    <hr>
+    <section>
+      <header>
+        <h2>Health Book</h2>
+      </header>
+      <!-- <button>
+        Add contact
+      </button> -->
+      <CreateContactForm />
+    </section>
+    <hr>
+    <section>
```

Comment:
did we decide not to include filtering in the beta version? I am fine with that, but if we are going that way I think we should include in the design revision doc.

Reply:
Yeah, I assumed the search (since it's an added feature/not really part of our concepts) would be for final. I can add a sentence about that in the design revision